doc/man3/OSSL_HTTP_REQ_CTX_free.pod

```diff
@@ -0,0 +1,70 @@
+=pod
+
+=head1 NAME
+
+OSSL_HTTP_REQ_CTX_free,
+OSSL_HTTP_REQ_CTX_set_max_response_length,
+OSSL_HTTP_REQ_CTX_add1_header,
+OSSL_HTTP_REQ_CTX_i2d
+- HTTP request functons
+
+=head1 SYNOPSIS
+
+ #include <openssl/http.h>
+
+ void OSSL_HTTP_REQ_CTX_free(OSSL_HTTP_REQ_CTX *rctx);
+
+ void OSSL_HTTP_REQ_CTX_set_max_response_length(OSSL_HTTP_REQ_CTX *rctx,
+                                                unsigned long len);
+
+ int OSSL_HTTP_REQ_CTX_add1_header(OSSL_HTTP_REQ_CTX *rctx,
+                                   const char *name, const char *value);
+
+ int OSSL_HTTP_REQ_CTX_i2d(OSSL_HTTP_REQ_CTX *rctx, const char *content_type,
+                           const ASN1_ITEM *it, ASN1_VALUE *req);
+
+=head1 DESCRIPTION
+
+OSSL_HTTP_REQ_CTX_free() frees up the OCSP context B<rctx>.
+
+OSSL_HTTP_REQ_CTX_set_max_response_length() sets the maximum response length
+for B<rctx> to B<len>. If the response exceeds this length an error occurs.
+If not set a default value of 100k is used.
+
+OSSL_HTTP_REQ_CTX_add1_header() adds header B<name> with value B<value> to the
+context B<rctx>. It can be called more than once to add multiple headers.
+It B<MUST> be called before any calls to OCSP_sendreq_nbio(). The B<req>
+parameter in the initial to OCSP_sendreq_new() call MUST be set to B<NULL> if
```

Comment:
`OCSP_sendreq_nbio()` should be `OSSL_HTTP_REQ_CTX_sendreq_d2i()`
`OCSP_sendreq_new()` should be `OSSL_HTTP_REQ_CTX_new()`

Reply:
Looks like you fixed this, too, in your commit.